### PR TITLE
Korean NumbersWithUnit specs localized

### PR DESCRIPTION
**Pull request by LionbridgeCS**
The requested extra cases for explicit USD, Korean units, and other Korean-specific cases will be added with the implementation in future PRs.

### CODE REVIEW

File: Specs/NumberWithUnit/Korean/CurrencyModel.json [hunk 1]
```diff
@@ -0,0 +1,1903 @@
...
+    "Input": "메릴랜드 주 몽고메리 카운티, 7천 5백만 달러 일반 채권, 시리즈 b,   1989년 매뉴팩처러스 하노버 트러스트에 의해 강화된 공공 개선  채권이다. ",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "7천 5백만",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "75000000",
+          "unit": "Dollar"
```
Comment (tellarin) [inline Specs/NumberWithUnit/Korean/CurrencyModel.json]: Are these general "dollar" or a term specific to US dollar? Sorry, I'm not familiar with Korean, so I'm double checking.

File: Specs/NumberWithUnit/Korean/DimensionModel.json [hunk 2]
```diff
@@ -0,0 +1,838 @@
+[
```
Comment (tellarin) [inline Specs/NumberWithUnit/Korean/DimensionModel.json]: Are there any Korea-specific units that are common enough the recognizers should support?

File: Specs/NumberWithUnit/Korean/CurrencyModel.json (hunk 1 above)
Comment (tellarin) [reply Specs/NumberWithUnit/Korean/CurrencyModel.json]: If all these cases are the general "dollar", can you add at least one case with the term specifically for USD too?

Comment (LionbridgeCS) [reply Specs/NumberWithUnit/Korean/CurrencyModel.json]: These are the general dollar term. The requested will be added with the implementation.

File: Specs/NumberWithUnit/Korean/DimensionModel.json (hunk 2 above)
Comment (LionbridgeCS) [reply Specs/NumberWithUnit/Korean/DimensionModel.json]: The Korean specific units will be added with the implementation.

**Pull request merged**